golem_messages/validators.py

```diff
@@ -19,3 +19,132 @@ def validate_integer(field_name, value):
             field=field_name,
             value=value,
         )
+
+
+def is_number_positive(value, value_type):
+    assert value_type in [int, float]
+    return isinstance(value, value_type) and value > 0
```

Comment:
Shouldn't this just throw *FieldError*?

Reply:
It shouldn't. Purpose of this function is just to give `true`/`false` answer. It is reused few times that's why I add it as separate function.

Reply:
Coming back to this. Please correct me if I'm wrong, but every time you use this function, you raise `FieldError` immediately when it returns `False`. That's why I insist it should just raise it, and be renamed to `verify_number_positive`.

Also every time except once, you check whether it's grater than *or equal* to zero. That's why this function should raise when `value < 0` and add additional check for *zero* in `validate_list_of_positive_integers`.

Reply:
after removing all parts related to LuxRender problem solved. So, Done.